src/buffer/out/textBuffer.cpp

```diff
@@ -2085,187 +1956,326 @@ std::wstring TextBuffer::GetPlainText(const til::point& start, const til::point&
     return text;
 }
 
+TextBuffer::CopyRequest TextBuffer::CopyRequest::FromConfig(const TextBuffer& buffer, const til::point& beg, const til::point& end, const bool singleLine, const bool blockSelection, const bool trimBlockSelection, const bool bufferCoordinates) noexcept

```

Comment:
We may want to inline this into the header so that a reader can see it all at once at the same time as `CopyRequest`

Selfishly: I couldn't originally tell _why_ this was different from the constructor itself 🙂